src/main/java/com/plaid/client/response/Account.java

```diff
@@ -1,9 +1,18 @@
 package com.plaid.client.response;
 
 public final class Account {
+
+  /**
+   * The verification status
+   */
+  public enum VerificationStatus {
+    pending_automatic_verification, pending_manual_verification, manually_verified
```

Comment:
oh no undercase enums

Reply:
looks like you're missing `automatically_verified`? 

(`automatically_verified` is listed in https://plaid.com/docs/#account-verification-statuses but not in https://plaid.com/docs/#onsuccess-callback)

Reply:
you're right @sloanesturz 🙈 just added this